packages/directives/repeat-click/index.ts

```diff
@@ -39,7 +47,7 @@ export const vRepeatClick: ObjectDirective<
       }
     }
 
-    el.addEventListener('mousedown', (evt: MouseEvent) => {
+    const onMouseDown = (evt: MouseEvent) => {
```

Comment:
There should be no need to change here; the bound event will be destroyed when the component is uninstalled.

What needs to be handled is to ensure that the timer is cleared when the component is uninstalled.